src/main.ts

```diff
@@ -91,10 +88,9 @@ export const main = async ({
       })
 
       if (result.published) {
-        setOutput('published', true)
-        setOutput('publishedPackages', result.publishedPackages)
-        exportVariable('PUBLISHED', true)
-        exportVariable('PUBLISHED_PACKAGES', result.publishedPackages)
+        const filePath = `${HOME}/published-packages.json`
+        fs.writeJsonSync(filePath, result.publishedPackages)
```

Comment:
The idea was that if we modified the code in 2 places, 

in main before the publish
```ts
setOutput('published', false)
setOutput('publishedPackages', [])
```

and in main if publish is successful,
```ts
setOutput('published', true)
setOutput('publishedPackages', result.publishedPackages)
```

it would be problematic since the first one (published false) will create a file in the directory that will be added to git in the `version` command if people will just upgrade their version. When I removed that it didn't really make sense anymore to have a published true either since it is implicitly `true` by the file being there.

So, I can make it an object of course, but should the object still contain `published`? And if so, at which points do we want to create this file; should the user be able to assume the file will always be there; also with the version command?

Reply:
> it would be problematic since the first one (published false) will create a file in the directory that will be added to git in the version command if people will just upgrade their version. When I removed that it didn't really make sense anymore to have a published true either since it is implicitly true by the file being there.

That's why I sad that this file should not be included in the project, instead, `$HOME` for example.